.github/workflows/ci_workflows.yml

```diff
@@ -134,3 +134,37 @@ jobs:
       test_command: pytest -p no:warnings --astropy-header -m "not hypothesis" -k "not test_data_out_of_range and not test_set_locale and not TestQuantityTyping" --pyargs astropy
       targets: |
         - cp311-manylinux_x86_64
+
+
+  test_limited_api_build:
```

Comment:
Is this allowed to fail or not? If not, I will have to add it to branch protection. Please advise. Thanks!